src/components/KBadge/KBadge.vue

```diff
@@ -378,6 +389,10 @@ watch(badgeText, () => {
   $KBadgeWarningBackground: var(--KBadgeWarningBackground, var(--yellow-100, color(yellow-100)));
   $KBadgeWarningColor: var(--KBadgeWarningColor, var(--yellow-600, color(yellow-600)));
   $KBadgeWarningButtonHoverColor: var(--KBadgeWarningButtonHoverColor, var(--yellow-200, color(yellow-200)));
+  // neutral appearance colors local variables
+  $KBadgeNeutralBackground: var(--KBadgeNeutralBackground, var(--grey-200, color(grey-200)));
+  $KBadgeNeutralColor: var(--KBadgeNeutralColor, var(--grey-500, color(grey-500)));
+  $KBadgeNeutralButtonHoverColor: var(--KBadgeNeutralButtonHoverColor, var(--grey-300, color(grey-300)));
```

Comment:
docs: can you add the new theming variables to the theming section of the docs page?

Reply:
@adamdehaven seems like we don't expose other variables that control appearances colors and looks like that's intentional. Either we list all of them in this section, or none - what do you think?

![Screen Shot 2023-06-06 at 9 21 00 AM](https://github.com/Kong/kongponents/assets/36751160/83b366a5-45e0-40d0-9238-44b44061fb81)

Reply:
Basically because we have both `backgroundColor` and `borderColor` props, this variables have become redundant. I will remove the ones I added.